.azure-devops/prbuild/mlz-pr-bicep-azurecloud-pipelines.yml

```diff
@@ -19,8 +19,68 @@ jobs:
   displayName: "PR Checks for Mission LZ Bicep on AzureCloud"
 
   steps:
-  - template: templates/az-what-if.yml
-    parameters:
-      ServiceConnectionName: $(CAzureConnection)
-      DeploymentName: $(bDeploymentName)
-      Location: $(CLocation)
+
+  - checkout: self
```

Comment:
Should this file be modified? It looks like this is the same work `mlz-pr-sbom-pipelines.yml` is doing.

We probably want the SBOM generation to occur in that other pipeline since its unrelated to the what-if check.

Reply:
SBOM pipeline configured as its own pipeline. Not sure whether merge did something differently

Reply:
https://github.com/Azure/missionlz/blob/brooke/sbom-pipeline/.azure-devops/prbuild/mlz-pr-sbom-pipelines.yml

Reply:
i  know the problem....my bad. I forgot to revert the code